UPGRADING.md

```diff
@@ -32,6 +32,8 @@ in case you have heavily used internal-api here are the most important the chang
 - Removed `Serializer::setSerializationContextFactory` and `Serializer::setDeserializationContextFactory`
 - Removed `Serializer::getMetadataFactory` 
 - As default now JSON preserve trailing zeros when serializing a float
+- When using a discriminator map, parent class should either be declared abstract, or included into the discriminator
```

Comment:
Looks a pretty common case extending a class that is not abstract... why enforcing that kind of constraint?

classical usecase: Abstract vehicle > concrete car > concrete electrical car

Reply:
But why would someone want to leave ConcreteCar not included into discriminator map then? If you can instantiate it, it means that you will potentially serialize/deserialize it, so you need to include it into the map.

Reply:
makes sese :+1: